docs/adding-to-project.md

```diff
@@ -21,8 +20,17 @@ To add BLT to a pre-existing Drupal project, do the following:
 
 1. Add BLT via composer:
 
-        rm -rf vendor composer.lock
```

Comment:
This was necessary when BLT was using composer merge plugin the 9.x branch, but no longer.